images/ceph/Dockerfile

```diff
@@ -26,5 +26,9 @@ COPY rook rookflex toolbox.sh set-ceph-debug-level /usr/local/bin/
 COPY ceph-monitoring /etc/ceph-monitoring
 COPY rook-external /etc/rook-external/
 COPY ceph-csv-templates /etc/ceph-csv-templates
+RUN useradd rook -u 2016 # 2016 is the UID of the rook user and also the year of the first commit in the project
+RUN mkdir -p /var/lib/rook
+RUN chown rook:rook /var/lib/rook
+USER 2016
```

Comment:
For upgrades, if they installed the operator in a prior release, now the upgrade will still run as root until they add the security context to run as 2016, right? So the upgrades should be fine.

Reply:
Yes, until the spec is changed to include the new `securityContext`, the user `root` will continue to be used. So yes it's fine.